packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3201,6 +3203,17 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     markNeedsSemanticsUpdate();
   }
 
+  /// If non-null, sets the [SemanticsNode.isPassword] semantic to the given
+  /// value.
+  bool get password => _password;
```

Comment:
this property sounds like it is the password, rather than a flag indicating the presence of the password. Maybe isPassword?

Reply:
The naming is currently in line with the naming choices for the other properties in this file (e.g. `button`, `header`, `textFields` etc.). Do you think the password property is different from the others and that's why we should name it `isPassword`? Or do you think that all the other properties should have been named with an is* prefix (e.g. `isHeader`, `isButton`, etc)?

Reply:
Renaming to `obscured`.